app/models/sac_cas/event/course.rb

```diff
@@ -193,7 +194,15 @@ def state_changed_from_draft_to_published?
     saved_change_to_attribute(:state) == ["created", "application_open"]
   end
 
-  def send_email_to_course_admin
+  def send_application_published_email
     Event::PublishedJob.new(self).enqueue!
   end
+
+  def state_changed_to_application_paused?
+    saved_change_to_attribute(:state)&.second == "application_paused"
+  end
+
+  def send_application_paused_email
+    Event::ApplicationPausedJob.new(self).enqueue!
```

Comment:
```suggestion
    Event::ApplicationPausedJob.new(self).enqueue! if groups.first.course_admin_email.present?
```
Damit wir nicht unnötig die DJ Queue füllen.